projects/packages/search/src/dashboard/components/pages/dashboard-page.jsx

```diff
@@ -147,13 +168,29 @@ export default function DashboardPage( { isLoading = false } ) {
 		);
 	};
 
+	const renderDonutMeterTestSection = () => {
+		return (
+			<div className="jp-search-record-meter jp-search-dashboard-wrap">
+				<div className="jp-search-dashboard-row">
+					<div className="lg-col-span-2 md-col-span-1 sm-col-span-0"></div>
+					<div className="jp-search-record-meter__content lg-col-span-8 md-col-span-6 sm-col-span-4">
+						<h2>DonutMeter layout testing</h2>
+						<DonutMeterWithLabels />
+					</div>
+					<div className="lg-col-span-2 md-col-span-1 sm-col-span-0"></div>
+				</div>
+			</div>
+		);
+	};
+
 	return (
 		<>
 			{ isPageLoading && <Loading /> }
 			{ ! isPageLoading && (
 				<div className="jp-search-dashboard-page">
 					{ renderHeader() }
 					{ renderMockedSearchInterface() }
+					{ renderDonutMeterTestSection() }
```

Comment:
Let's not do this. There's no point in using functions called `renderSomething` when components are actually functions that just return what's gonna be rendered. If you go with this approach this function is not visible for the React Dev Tools as a component. 

Go for a name like `const DonutMeterTestSection = ()...` and use it like this `<DonutMeterTestSection />`

In this particular example I understand there were `renderHeader` and `renderMockedSearchInterface` not introduced in the PR, but let's just do it properly with `DonutMeterTestSection`

Reply:
Thanks for the feedback @oskosk! Much appreciated!

In this case, I'm building out `DonutMeterWithLabels` as the actual component but I understand what you're getting at here. Maybe moving this code block into a "layout" component is the best option here? I'll give that a try as I continue to build this out.

Reply:
Thanks for the feedback @oskosk! Much appreciated!

In this case, I'm building out DonutMeterWithLabels as the actual component but I understand what you're getting at here. Maybe moving this code block into a "layout" component is the best option here? I'll give that a try as I continue to build this out.